motoko/README.md

```diff
@@ -2,6 +2,49 @@
 
 A library for making requests to the LLM canister on the Internet Computer.
 
+## Local Development Setup
+
+Before using this library in local development, you need to set up the LLM canister dependency:
+
+### Prerequisites
+- [DFX](https://internetcomputer.org/docs/building-apps/getting-started/install) installed
+- [Ollama](https://ollama.com/) installed and running locally
+
+### Setup Steps
+
+1. **Start Ollama** (required for local development):
+```bash
+# Start the Ollama server
+ollama serve
+
+# Download the required model (one-time setup)
+ollama run llama3.1:8b
+```
+
+2. **Add LLM canister to your dfx.json**:
+```json
+{
+  "canisters": {
+    "llm": {
+      "type": "pull",
+      "id": "w36hm-eqaaa-aaaal-qr76a-cai"
+    },
+    "your-canister": {
+      "dependencies": ["llm"],
+      "type": "motoko"
+    }
+  }
+}
```

Comment:
should we provide the [alternative way](https://github.com/dfinity/examples/blob/e30c25d01cd4046f507489f1da04bb474896fea4/motoko/llm_chatbot/dfx.json#L17) without using `pull` as done in the `examples`as well? if so, we would need to provide an alternative to the steps below as well